Cargo.toml

```diff
@@ -25,13 +25,11 @@ aws-smithy-types = "0.39.0"
 aws-smithy-types-convert = { version = "0.39.0", features = ["convert-chrono"] }
 aws-types = "0.9.0"
 bytes = "1.1.0"
-datafusion-data-access = { version = "8.0.0" }
+datafusion = { version = "9.0.0" }
```

Comment:
Its so hard to keep up with these API changes!

Reply:
Agreed. For now it's still manageable.